src/nxdoc/nuxeo-server/administration/configuration-parameters-index-nuxeoconf.md

```diff
@@ -2365,6 +2365,18 @@ Property to override the default Content Security Policy.
 
 * * *
 
+#### `org.nuxeo.web.ui.trustedDomains`
+
+Allow cross repository navigation to specified URLs. If left blank or no property defined in configuration file it will allow to redirect all URLs.
```

Comment:
If left blank or no property defined in configuration file, then it will allow to redirect all URLs.

Reply:
incorporated